builtin/providers/circonus/data_source_circonus_account.go

```diff
@@ -0,0 +1,269 @@
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						accountEmailAttr: &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: accountDescription[accountEmailAttr],
+						},
+						accountRoleAttr: &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: accountDescription[accountRoleAttr],
+						},
+					},
+				},
+			},
+			accountNameAttr: &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: accountDescription[accountNameAttr],
+			},
+			accountOwnerAttr: &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: accountDescription[accountOwnerAttr],
+			},
+			accountStateProvAttr: &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: accountDescription[accountStateProvAttr],
+			},
+			accountTimezoneAttr: &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: accountDescription[accountTimezoneAttr],
+			},
+			accountUIBaseURLAttr: &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: accountDescription[accountUIBaseURLAttr],
+			},
+			accountUsageAttr: &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: accountDescription[accountUsageAttr],
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						accountLimitAttr: &schema.Schema{
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: accountDescription[accountLimitAttr],
+						},
+						accountTypeAttr: &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: accountDescription[accountTypeAttr],
+						},
+						accountUsedAttr: &schema.Schema{
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: accountDescription[accountUsedAttr],
+						},
+					},
+				},
+			},
+			accountUsersAttr: &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: accountDescription[accountUsersAttr],
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						accountUserIDAttr: &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: accountDescription[accountUserIDAttr],
+						},
+						accountRoleAttr: &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: accountDescription[accountRoleAttr],
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceCirconusAccountRead(d *schema.ResourceData, meta interface{}) error {
+	c := meta.(*providerContext)
+
+	var cid string
+
+	var a *api.Account
+	var err error
+	if v, ok := d.GetOk(accountIDAttr); ok {
+		cid = v.(string)
+	}
+
+	if v, ok := d.GetOk(accountCurrentAttr); ok {
```

Comment:
ok, after speaking to @sean- - my comment here isn't correct, this isn't being added to a struct to therefore, we don't need a default value - therefore, this is correct!